Task 2/Task 2.1/Task 2.1.2/Polygons/Quadrangle.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace Task_2._1._2.Polygons
+{
+    class Quadrangle : Polygon
+    {
+        private enum QuadrangleType
+        {
+            Square,
+            Rectangle,
+            other
```

Comment:
1. should be UpperCase
2. You can call it Quadrangle (or Quad, like in 3D Editors)